frontend/components/questions/question-create.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import { Question } from "@/lib/schemas/question-schema";
+import QuestionForm from "@/components/questions/question-form";
+import { useAuth } from "@/app/auth/auth-context";
+import { useRouter } from "next/navigation";
+import { useToast } from "@/components/hooks/use-toast";
+
+export default function QuestionCreate() {
+  const auth = useAuth();
+  const router = useRouter();
+  const { toast } = useToast();
+
+  const handleCreate = async (newQuestion: Question) => {
```

Comment:
Might need to use my schema after my changes are pushed in.

Reply:
![image](https://github.com/user-attachments/assets/b39c30e9-db6c-416d-8cfa-dda6bddce8b7)
So you can do something like this